src/services/_exampleService.js

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import { Observable } from 'rxjs';
+import { toExampleItemModel, toExampleItemsModel } from './models';
+
+/** Normally, you'll need to define the endpoint URL.
+ * See app.config.js to add a new service URL.
+ *
+ * For this example, we'll just hardcode sample data to be returned instead
+ * of making an actual service call. See the other service files for examples.
+ */
+//const ENDPOINT = Config.serviceUrls.example;
+
+/** Contains methods for calling the example service */
+export class ExampleService {
+
+  /** Returns an example item */
+  static getExampleItem(id) {
+    return Observable.of(
```

Comment:
No need for the double quotes

`return Observable.of({ ID: id, Description: "This is an example item." })`